wowchemy/layouts/partials/book_layout.html

```diff
@@ -24,6 +27,10 @@
       <article class="article">
 
         <div class="docs-article-container">
+          {{ if $breadcrumb }}
+            {{ partial "breadcrumb" $page }}
```

Comment:
Have you run this code? Where is $page set? It appears to be named $current_page?

Reply:
You're absolutely right. Must have forgotten to carry the correct name over from my test repo. Fixed!